cmd/integration_test/build/buildspecs/quick-test-eks-a-cli.yml

```diff
@@ -130,7 +130,11 @@ phases:
           BUNDLES_OVERRIDE=true
         fi
       - SKIPPED_TESTS=$(yq e ".skipped_tests | @csv" ${CODEBUILD_SRC_DIR}/test/e2e/SKIPPED_TESTS.yaml)
-      - QUICK_TESTS="($(yq e '.quick_tests | join("|")' ./test/e2e/QUICK_TESTS.yaml))"
+      - TESTS="${TESTS:-$(yq e '.quick_tests | join("|")' ./test/e2e/QUICK_TESTS.yaml)}"
```

Comment:
Great idea modify test case through environment variable! Looks like this line can be removed?

Reply:
good catch!
changed